django/utils/crypto.py

```diff
@@ -47,14 +47,12 @@ def salted_hmac(key_salt, value, secret=None, *, algorithm='sha1'):
 
 
 NOT_PROVIDED = object()  # RemovedInDjango40Warning.
+RANDOM_STRING_CHARS = 'abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789'
```

Comment:
couldn't we use the following code here?
```
import string
RANDOM_STRING_CHARS = string.ascii_letters + string.digits
```

Reply:
The length of alphabet is important in few places, that's why I would prefer the current form.

Reply:
The code I proposed sets the same content in the variable with the same length. It's not a major change, just a small improvement. IMO